src/core/Table/filters/DateRangeFilter/DateRangeFilter.tsx

```diff
@@ -0,0 +1,109 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../../../utils/hooks/useTheme';
+import {
+  FilterButtonBar,
+  FilterButtonBarTranslation,
+} from '../FilterButtonBar';
+import { BaseFilter } from '../BaseFilter';
+import { TableFilterProps } from '../types';
+import '@itwin/itwinui-css/css/table.css';
+import DatePickerInput from './DatePickerInput';
+
+export type DateRangeTranslation = {
+  from: string;
+  to: string;
+};
+
+const defaultStrings: DateRangeTranslation = {
+  from: 'From',
+  to: 'To',
+};
+
+const defaultFormatDate = (date: Date) => {
+  const formatter = new Intl.DateTimeFormat('en-us', {
```

Comment:
I think `en-us` is not necessary and it can just be `undefined` to get the browser default

Reply:
If it takes user browser language it may break stuff - parser and placeholder may be wrong. As we use English in all places, I left for the developer to handle this.